internal/saucecloud/imagerunner.go

```diff
@@ -364,14 +366,19 @@ func (r *ImgRunner) collectResults(results chan execResult, expected int) bool {
 		}
 
 		r.PrintResult(res)
-		if !r.Project.LiveLogs {
-			// only print logs if live logs are disabled
-			r.PrintLogs(res.runID, res.name)
-		}
-		files := r.DownloadArtifacts(res.runID, res.name, res.status, res.err != nil)
+
 		var artifacts []report.Artifact
-		for _, f := range files {
-			artifacts = append(artifacts, report.Artifact{FilePath: f})
+		if res.assetsStatus == imagerunner.RunnerAssetStateErrored {
+			log.Warn().Msg("Logs and artifacts could not be uploaded due to an error, skipping download")
```

Comment:
(optional) Just a wee bit more succinct
```suggestion
			log.Warn().Msg("Logs and artifacts are not available due to an error error.")
```